test/test_markov_approx.jl

```diff
@@ -31,7 +31,7 @@
     X = simulate(mc, 100_000)
     mc2 = estimate_mc_discrete(X, mc.state_values)
     @test isequal(mc.state_values, mc2.state_values)
-    @test isapprox(mc.p, mc2.p; atol=1e-2)
+    @error norm(mc.p - mc2.p)
```

Comment:
Can you explain what this change was for?

Thanks!

Reply:
This test is failing locally.(The norm of the difference is about 0.01)

But I don't think this is related to this PR.

Reply:
This test is also failing on the current release.

Reply:
@sglyon I reverted that line and removed the switch to Github Actions in this PR (this is now in #272)

Please let me know what I can do to facilitate the review for you.